packages/manager/modules/exchange/src/account/mfa/bulk-create.html

```diff
@@ -0,0 +1 @@
+<exchange-account-mfa-create></exchange-account-mfa-create>
```

Comment:
Isn't it possible to just pass the component to a `ui-view` or something like that ?

Reply:
Invoked with some `setAction` code. I haven't refactor the routing.